### PR TITLE
Auto-import DNS records article

### PR DESCRIPTION
**Pull request by sbastn**
This PR introduces a new support article for our recent auto-import DNS feature.

### CODE REVIEW

File: content/articles/auto-import-dns.markdown [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+excerpt: Avoid downtime when transferring or hosting your domain with us by auto-importing your DNS records
+categories:
+- DNS
+---
+
+# Auto-importing DNS records
+
+Auto-importing DNS records can help you reduce downtime on your domain transfer or when hosting the DNS for your domain with us.
+
+Instead of entering manually DNS records or importing them in BIND format (not all domain providers let you export your records in this standard format), we scan existing records and import them for you.
```
[the remark this answers is not in the document]
Comment (sbastn) [reply content/articles/auto-import-dns.markdown]: ✅ Thanks for the suggestion

File: content/articles/auto-import-dns.markdown [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+We currently can import the following DNS record types:
+- A
+- AAAA
+- CNAME
+- MX
+- TXT
+
+<note>
+This feature is available on Personal, Professional, Business, and Reseller plans
```
Comment (jacegu) [inline content/articles/auto-import-dns.markdown]: You may want to look at how we are expressing this in other articles (Like LE ones). This doesn't make me think that the other plans are Legacy ones that we want to discontinue.

File: content/articles/auto-import-dns.markdown [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+</note>
+
+We prompt you to auto-import DNS records from the following flows:
```
Comment (jacegu) [inline content/articles/auto-import-dns.markdown]: Flows? Is this article only for UX connoisseurs? 

> We prompt you to auto-import DNS records when:
> - A domain transfer is about to start
> - Adding a domain for DNS management

File: content/articles/auto-import-dns.markdown [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+- When a domain transfer is about to start
+- When adding a domain for DNS management
+
+You can also auto-import the zone for your domain at any time by clicking on <label>Auto-import DNS</label> from the record editor.
+
+## Overview
+
+We scan the existing records for your domain and let you import them all at once.
+
+We automatically look for the most common subdomain like "www", "blog", and "email", but you can add custom subdomains at the bottom of the form.
```
Comment (jacegu) [inline content/articles/auto-import-dns.markdown]: most common subdomain**s**

**Pull request merged**